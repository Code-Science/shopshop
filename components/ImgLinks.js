import React from 'react';
import Image from 'next/image';
import Link from 'next/link';
import styles from '../styles/components/ImgLinks.module.css';

function ImgLinks() {
  return (
    <div className={styles.bottomCollection}>
      <Link href="/collection/women/clothes">
        <a className={styles.image}>
          jhghjgj
          <Image
            src="/images/bottomPic1.jpg"
            alt="home banner"
            objectFit="cover"
            layout="fill"
          />
        </a>
      </Link>
      <Link href="/collection/women/accessories">
        <a className={styles.image}>
          jhghjgj
          <Image
            src="/images/bottomPic2.jpg"
            alt="home banner"
            objectFit="cover"
            layout="fill"
          />
        </a>
      </Link>
      <Link href="/collection/men/clothes">
        <a className={styles.image}>
          jhghjgj
          <Image
            src="/images/bottomPic3.jpg"
            alt="home banner"
            objectFit="cover"
            layout="fill"
          />
        </a>
      </Link>
      <Link href="/collection/women/accessories">
        <a className={styles.image}>
          jhghjgj
          <Image
            src="/images/bottomPic4.jpg"
            alt="home banner"
            objectFit="cover"
            layout="fill"
          />
        </a>
      </Link>
      <Link href="/collection/kids/clothes">
        <a className={styles.image}>
          jhghjgj
          <Image
            src="/images/bottomPic5.jpg"
            alt="home banner"
            objectFit="cover"
            layout="fill"
          />
        </a>
      </Link>
    </div>
  );
}

export default ImgLinks;
