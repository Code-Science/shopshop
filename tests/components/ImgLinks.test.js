import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import ImgLinks from '../../components/ImgLinks';

jest.mock('../../store/StateProvider');

configure({ adapter: new Adapter() });

describe('ImgLinks Component', () => {
  const wrapper = shallow(<ImgLinks />);

  it('expect to render ImgLinks component', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
