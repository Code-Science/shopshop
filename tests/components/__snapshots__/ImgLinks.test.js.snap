// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImgLinks Component snapshot test 1`] = `
<div
  className="bottomCollection"
>
  <Link
    href="/collection/women/clothes"
  >
    <a
      className="image"
    >
      jhghjgj
      <Image
        alt="home banner"
        layout="fill"
        objectFit="cover"
        src="/images/bottomPic1.jpg"
      />
    </a>
  </Link>
  <Link
    href="/collection/women/accessories"
  >
    <a
      className="image"
    >
      jhghjgj
      <Image
        alt="home banner"
        layout="fill"
        objectFit="cover"
        src="/images/bottomPic2.jpg"
      />
    </a>
  </Link>
  <Link
    href="/collection/men/clothes"
  >
    <a
      className="image"
    >
      jhghjgj
      <Image
        alt="home banner"
        layout="fill"
        objectFit="cover"
        src="/images/bottomPic3.jpg"
      />
    </a>
  </Link>
  <Link
    href="/collection/women/accessories"
  >
    <a
      className="image"
    >
      jhghjgj
      <Image
        alt="home banner"
        layout="fill"
        objectFit="cover"
        src="/images/bottomPic4.jpg"
      />
    </a>
  </Link>
  <Link
    href="/collection/kids/clothes"
  >
    <a
      className="image"
    >
      jhghjgj
      <Image
        alt="home banner"
        layout="fill"
        objectFit="cover"
        src="/images/bottomPic5.jpg"
      />
    </a>
  </Link>
</div>
`;
